app/assets/stylesheets/partials/_alerts.css.scss

```diff
@@ -22,3 +22,25 @@ ul.govuk-list.govuk-error-summary__list li {
   @include govuk-font($size: 19, $weight: bold)
   color:  $govuk-error-colour;
 }
+
+.environment-info-wrapper {
```

Comment:
Shall we make the banner's font match the rest of the site here? Can do that with:

`@include govuk-font($size: 19, $weight: normal);`

Reply:
Are you happy to make this change?